funny.txt

```diff
@@ -0,0 +1 @@
+I don’t trust stairs. They’re always up to something.
```

Comment:
can you please add an explanation why the joke is funny?

Reply:
Done in commit 2f07b12 "Explanation".